src/test/java/hudson/plugins/junitattachments/AttachmentPublisherTest.java

```diff
@@ -176,7 +176,7 @@ private void runBuildAndAssertAttachmentsExist(String className, String[] expect
     }
 
     // Asserts that, for the given TestResult, the given attachments exist
-    private static void assertAttachmentsExist(TestResult result, String[] expectedFiles) {
+    protected static void assertAttachmentsExist(TestResult result, String[] expectedFiles) {
```

Comment:
:ant: `protected static` makes little sense except in a few special cases. If you wish to expose these methods to other test suites, just delete `private` (use default package access).